plasma_framework/contracts/src/exits/payment/PaymentInFlightExitable.sol

```diff
@@ -286,47 +290,22 @@ contract PaymentInFlightExitable is
         return amountIn;
     }
 
-    function startExit(StartExitData memory startExitData, bool isCanonical) private {
-        UtxoPosLib.UtxoPos memory youngestInputUtxo = getYoungestInputUtxo(startExitData.inputUtxosPos);
-
+    function startExit(StartExitData memory startExitData) private {
         PaymentExitDataModel.InFlightExit storage ife = inFlightExits[startExitData.exitId];
         ife.bondOwner = msg.sender;
         ife.exitStartTimestamp = block.timestamp;
-        ife.exitPriority = getInFlightExitPriority(startExitData.exitId, youngestInputUtxo.value, block.timestamp);
         setInFlightExitInputs(ife, startExitData.inputTxs, startExitData.inputUtxosPos);
-        if (!isCanonical) {
-            setNonCanonical(ife);
-        }
-    }
-
-    function getYoungestInputUtxo(UtxoPosLib.UtxoPos[] memory inputUtxosPos) private pure returns (UtxoPosLib.UtxoPos memory) {
```

Comment:
ah...sorry, i think I mislead a bit. I don't think we need `getInFlightExitPriority` but we would need this `youngestestInputUtxo`. We would pass in `exitableAt` and `txPos` to compose the priority of the queue. I was wrong that `exitableAt` would be enough. However, even to get the exitableAt, we would need the `youngestestInputUtxo`.

Reply:
done